src/libs/actions/Policy/Tag.ts

```diff
@@ -594,7 +594,6 @@ function setPolicyTagsRequired(policyID: string, requiresTag: boolean, tagListIn
                 value: {
                     [policyTag.name]: {
                         required: requiresTag,
-                        errors: null,
```

Comment:
Readd this one. You should remove the one in `successData` data instead

Reply:
https://github.com/Expensify/App/pull/42972/commits/276f6e3ccab9b61184923c4c8688b4f537c4e701